packages/xstate-lit/src/useActorRef.ts

```diff
@@ -0,0 +1,9 @@
+import { Actor, ActorOptions, AnyActorLogic, createActor } from 'xstate';
+
+export const useActorRef = <TMachine extends AnyActorLogic>(
+  logic: TMachine,
+  options?: ActorOptions<TMachine>
+): Actor<TMachine> => {
+  const actorRef = createActor(logic, options);
+  return actorRef;
+};
```

Comment:
it's just a wrapper around `createActor` - do you even need it here? couldn't u just call `createActor` directly in ur `UseMachine`?